w_common/test/unit/browser/cache/least_recently_used_strategy_test.dart

```diff
@@ -73,7 +73,12 @@ void main() {
         // release expected item
         var release = cache.release(expectedId);
 
-        await Future.wait([firstGet, remove, secondGet, release]);
+        await Future.wait([
+          firstGet!.then((value) => value),
```

Comment:
I'm not sure I see the value of the .then((value) => value) I think the migrator did a weird thing here.